<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>0</number>
     </property>
     <widget class="QWidget" name="FPGApage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>547</width>
        <height>156</height>
       </rect>
      </property>
      <attribute name="label">
       <string>FPGA</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Bitfile</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Device Identifier</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Instrument</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBoxAutoUpload">
           <property name="text">
            <string>Automatically upload bitfile</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="showOnStartupCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Show on startup</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButtonScan">
           <property name="text">
            <string>Scan</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="uploadButton">
           <property name="text">
            <string>Upload</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QPushButton" name="renameButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Rename</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QToolButton" name="toolButtonOpenBitfile">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="Experiment.qrc">
             <normaloff>:/openicon/icons/document-open-5.png</normaloff>:/openicon/icons/document-open-5.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="comboBoxBitfiles"/>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="comboBoxInstruments">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="identifierEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>Dialog</receiver>
   <slot>apply(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply(QAbstractButton*)</slot>
 </slots>
</ui>
